lib/bson/parser/deserializer.js

```diff
@@ -384,7 +392,8 @@ var deserializeObject = function(buffer, index, options, isArray) {
         buffer[index + stringSize - 1] !== 0
       )
```

Comment:
throw new Error('' + string + ' not a valid Decimal128 string');

Reply:
huh?

Reply:
I have no idea how this got here...